distod/src/main/scala/com/github/codelionx/distod/actors/partitionMgmt/PartitionManager.scala

```diff
@@ -107,21 +115,14 @@ class PartitionManager(
       )
       partitions = CompactingPartitionMap(compactionSettings).from(singletonPartitions)
       stash.unstashAll(
-        behavior(attributes, PendingJobMap.empty)
+        behavior(attributes)
       )
     } else {
       initialize(attributes, singletonPartitions)
     }
 
-  private def behavior(
-      attributes: Seq[Int],
-      pendingJobs: PendingJobMap[CandidateSet, PendingResponse]
-  ): Behavior[PartitionCommand] = {
-    def next(
-        _attributes: Seq[Int] = attributes,
-        _pendingJobs: PendingJobMap[CandidateSet, PendingResponse] = pendingJobs,
-    ): Behavior[PartitionCommand] =
-      behavior(_attributes, _pendingJobs)
+  private def behavior(attributes: Seq[Int]): Behavior[PartitionCommand] = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/CodeLionX/distod/pullRequest?prid=5048550)